### PR TITLE
Add styles to Toggle

### LINKED ISSUE #984: [Feature] Add styles to Toggle component
Continuation from #963. Add styling to `Toggle` component so it can have different colours applied to it, etc.

### PR DESCRIPTION
**Pull request by Coteh**
Closes #984. Adds blue, black, red, and green colors to `Toggle`, and adds shadow option as well.

~Also updates `Button`'s storybook to add a green style that was in its SCSS module but not previously included in its storybook.~ edit: Split into #991

### CODE REVIEW

File: frontend/src/component/styles/shadow.scss [hunk 1]
```diff
@@ -9,3 +9,5 @@ $shadow-floating-box: 0 4px 6px 0 rgba(32, 33, 36, 0.28);
 $shadow-header: 0 3px 6px color-transparent-black(0.16),
   0 3px 6px rgba(0, 0, 0, 0.23);
 $shadow-drawer: 0 0 6px color-transparent-black(0.2);
+$shadow-toggle: 0 1px 3px color-transparent-black(0.12),
```
Comment (Coteh) [inline frontend/src/component/styles/shadow.scss:12]: Currently reusing `Button`'s shadow. Seems like it'd work fine for `Toggle`'s shadow too, but having it as its own SCSS variable will allow it to be updated independently of `Button`'s.

File: frontend/src/component/ui/Toggle.tsx [hunk 2]
```diff
@@ -44,19 +44,26 @@ export class Toggle extends Component<Props, State> {
   render() {
     const { enabled } = this.state;
     return (
-      <div className={styles.toggle}>
+      <div
+        className={`${withCSSModule(this.props.styles, styles)} ${
```
Comment (byliuyang) [inline frontend/src/component/ui/Toggle.tsx:48]: Is this needed to style the `Toggle`?

File: frontend/src/component/ui/Toggle.tsx [hunk 3]
```diff
@@ -44,19 +44,26 @@ export class Toggle extends Component<Props, State> {
...
             [styles.active]: enabled
-          })}
+          })}`}
           onClick={this.handleClick}
         >
           <div
-            className={classNames({
+            className={`${withCSSModule(
+              this.props.styles,
+              styles
```
Comment (byliuyang) [inline frontend/src/component/ui/Toggle.tsx:62]: Is this needed to style the `Toggle`?

File: story/src/0-Button.stories.tsx [hunk 4]
```diff
@@ -40,6 +40,14 @@ export const red = () => {
...
 };
 
+export const green = () => {
+  return (
+    <Button styles={['green']} onClick={action('click')}>
+      {text('Label', 'Button')}
+    </Button>
+  );
+};
+
```
Comment (byliuyang) [inline story/src/0-Button.stories.tsx:50]: Let's add this in a separate PR!

File: frontend/src/component/ui/Toggle.tsx (hunk 2 above)
Comment (Coteh) [reply frontend/src/component/ui/Toggle.tsx:48]: No. Only background is styled right now. Should this be removed?

File: frontend/src/component/ui/Toggle.tsx (hunk 3 above)
Comment (Coteh) [reply frontend/src/component/ui/Toggle.tsx:62]: No. Only background is styled right now. Should this be removed?

File: frontend/src/component/ui/Toggle.tsx (hunk 2 above)
Comment (byliuyang) [reply frontend/src/component/ui/Toggle.tsx:48]: Yes, please. We will add it back when needed!

File: frontend/src/component/ui/Toggle.tsx (hunk 3 above)
Comment (byliuyang) [reply frontend/src/component/ui/Toggle.tsx:62]: Yes, please. We will add it back when needed!

**Pull request merged**